packages/api-nest/test/helpers/create-e2e-app.ts

```
@@ -0,0 +1,159 @@
+ * @param customize - Function to customize the TestingModuleBuilder
+ * @returns Initialized NestJS application ready for testing
+ *
+ * @example
+ * ```typescript
+ * const app = await createE2EAppWithCustomization(builder =>
+ *   builder
+ *     .overrideProvider(MyService)
+ *     .useValue(mockService)
+ * )
+ * ```
+ */
```

ESLint

[error] Missing JSDoc return type.

(valid-jsdoc)